src/sqlfluff/dialects/dialect_clickhouse_keywords.py

```diff
@@ -171,4 +172,13 @@
     "WHERE",
     "WITH",
     "YEAR",
+    # Database Engine
+    "Atomic",
```

Comment:
Can we please follow the convention of keywords being uppercase?

Reply:
Fixed in commit `f56765e`.